interview-questions.md

```diff
@@ -7,41 +7,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: I think instance variable is a variable you use to call on a method. Difference between this variable and other variables such as javascript is that is javascript you use the variable as a placeholder to pass down information.
 
-  Researched answer:
+  Researched answer: Instance variable starts with an @ symbol meaning it belongs to a class.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: A block in Ruby is a block of code...?
 
-  Researched answer:
+  Researched answer: Blocks are anonymous functions in Ruby that are created with do/end keywords.
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: implicit return means to define the function.
```

Comment:
This is a good moment where you would want to work through a longer answer as it might help you get the right one. Every Answer in an interview should **attempt** to answer the big six questions: Who , What, When, Where, Why, & How?